onyo/commands/get.py

```diff
@@ -0,0 +1,145 @@
+import logging
+import re
+import sys
+from collections import Counter
+from typing import Generator, Union
+from pathlib import Path
+
+from rich import box
+from rich.console import Console
+from rich.table import Table
+
+from onyo import Filter, Repo, OnyoInvalidRepoError, OnyoInvalidFilterError
+
+logging.basicConfig()
+log = logging.getLogger('onyo')
+
+
+def natural_sort(
+        assets: list[tuple[Path, dict[str, str]]],
+        keys: Union[list, None] = None, reverse: bool = False) -> list:
+    """
+    Sort the output of `Repo.get()` by a given list of `keys`.
```

Comment:
Good points! I've expanded upon the docstring, as: "Sort the output of `Repo.get()` by a given list of `keys` or by the path of the `assets` if no `keys` are provided."

I have also made it so it uses `if keys:` rather than `if not keys`.